rust/otap-dataflow/crates/otap/src/attributes_processor.rs

```diff
@@ -194,13 +194,13 @@ impl local::Processor<OtapPdata> for AttributesProcessor {
                 }
 
                 let signal = pdata.signal_type();
-                let mut records: OtapArrowRecords = pdata.try_into()?;
+                let mut records: OtapArrowRecords = pdata.take_payload().try_into()?;
```

Comment:
See -- the "pdata" object is consumed. I made the pdata mut so that I could take the payload and consume it, leaving me a valid context. On line 203 below I should be using that context.